Data Race detected

**What happened**:
During the test run a data race was detected and should be investigated:

```bash
WARNING: DATA RACE
Write at 0x000003ec1798 by goroutine 58:
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh.SetKeepAliveInterval()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/export_test.go:40 +0x4a4
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh_test.glob..func1.4.5()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/ssh_test.go:379 +0x497
  github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/leafnodes/runner.go:113 +0x10c
  github.com/onsi/ginkgo/internal/leafnodes.(*runner).run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/leafnodes/runner.go:64 +0x157
  github.com/onsi/ginkgo/internal/leafnodes.(*ItNode).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/leafnodes/it_node.go:26 +0x94
  github.com/onsi/ginkgo/internal/spec.(*Spec).runSample()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/spec/spec.go:215 +0x359
  github.com/onsi/ginkgo/internal/spec.(*Spec).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/spec/spec.go:138 +0x1b0
  github.com/onsi/ginkgo/internal/specrunner.(*SpecRunner).runSpec()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/specrunner/spec_runner.go:200 +0x156
  github.com/onsi/ginkgo/internal/specrunner.(*SpecRunner).runSpecs()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/specrunner/spec_runner.go:170 +0x255
  github.com/onsi/ginkgo/internal/specrunner.(*SpecRunner).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/specrunner/spec_runner.go:66 +0x138
  github.com/onsi/ginkgo/internal/suite.(*Suite).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/suite/suite.go:79 +0x6f5
  github.com/onsi/ginkgo.runSpecsWithCustomReporters()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/ginkgo_dsl.go:245 +0x217
  github.com/onsi/ginkgo.RunSpecs()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/ginkgo_dsl.go:220 +0x267
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh_test.TestCommand()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/ssh_suite_test.go:29 +0xe4
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1259 +0x22f
  testing.(*T).Run·dwrap·21()
      /usr/local/go/src/testing/testing.go:1306 +0x47

Previous read at 0x000003ec1798 by goroutine 84:
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh.keepBastionAlive()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/options.go:950 +0x8a
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh.(*SSHOptions).Run·dwrap·3()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/options.go:527 +0x99

Goroutine 58 (running) created at:
  testing.(*T).Run()
      /usr/local/go/src/testing/testing.go:1306 +0x726
  testing.runTests.func1()
      /usr/local/go/src/testing/testing.go:1598 +0x99
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1259 +0x22f
  testing.runTests()
      /usr/local/go/src/testing/testing.go:1596 +0x7ca
  testing.(*M).Run()
      /usr/local/go/src/testing/testing.go:1504 +0x9d1
  main.main()
      _testmain.go:95 +0x324

Goroutine 84 (finished) created at:
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh.(*SSHOptions).Run()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/options.go:527 +0x1399
  github.com/gardener/gardenctl-v2/pkg/cmd/base.WrapRunE.func1()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/base/options.go:57 +0x1bd
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh_test.glob..func1.4.3()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/ssh_test.go:275 +0x4aa
  github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/leafnodes/runner.go:113 +0x10c
  github.com/onsi/ginkgo/internal/leafnodes.(*runner).run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/leafnodes/runner.go:64 +0x157
  github.com/onsi/ginkgo/internal/leafnodes.(*ItNode).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/leafnodes/it_node.go:26 +0x94
  github.com/onsi/ginkgo/internal/spec.(*Spec).runSample()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/spec/spec.go:215 +0x359
  github.com/onsi/ginkgo/internal/spec.(*Spec).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/spec/spec.go:138 +0x1b0
  github.com/onsi/ginkgo/internal/specrunner.(*SpecRunner).runSpec()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/specrunner/spec_runner.go:200 +0x156
  github.com/onsi/ginkgo/internal/specrunner.(*SpecRunner).runSpecs()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/specrunner/spec_runner.go:170 +0x255
  github.com/onsi/ginkgo/internal/specrunner.(*SpecRunner).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/specrunner/spec_runner.go:66 +0x138
  github.com/onsi/ginkgo/internal/suite.(*Suite).Run()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/internal/suite/suite.go:79 +0x6f5
  github.com/onsi/ginkgo.runSpecsWithCustomReporters()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/ginkgo_dsl.go:245 +0x217
  github.com/onsi/ginkgo.RunSpecs()
      /go/pkg/mod/github.com/onsi/ginkgo@v1.16.5/ginkgo_dsl.go:220 +0x267
  github.com/gardener/gardenctl-v2/pkg/cmd/ssh_test.TestCommand()
      /tmp/build/a94a8fe5/pull-request-gardener.gardenctl-v2-pr.master/pkg/cmd/ssh/ssh_suite_test.go:29 +0xe4
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1259 +0x22f
  testing.(*T).Run·dwrap·21()
      /usr/local/go/src/testing/testing.go:1306 +0x47
==================
••
Ran 11 of 11 Specs in 5.485 seconds
SUCCESS! -- 11 Passed | 0 Failed | 0 Pending | 0 Skipped
--- FAIL: TestCommand (5.50s)
    testing.go:1152: race detected during execution of test
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know**:

**Environment**:

--- a/pkg/cmd/ssh/export_test.go
+++ b/pkg/cmd/ssh/export_test.go
@@ -37,5 +37,8 @@ func SetPollBastionStatusInterval(d time.Duration) {
 }
 
 func SetKeepAliveInterval(d time.Duration) {
+	keepAliveIntervalMutex.Lock()
+	defer keepAliveIntervalMutex.Unlock()
+
 	keepAliveInterval = d
 }

--- a/pkg/cmd/ssh/options.go
+++ b/pkg/cmd/ssh/options.go
@@ -23,6 +23,7 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"sync"
 	"time"
 
 	gardencorev1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
@@ -64,7 +65,8 @@ const (
 var (
 	// keepAliveInterval is the interval in which bastions should be given the
 	// keep-alive annotation to prolong their lifetime.
-	keepAliveInterval = 3 * time.Minute
+	keepAliveInterval      = 3 * time.Minute
+	keepAliveIntervalMutex sync.RWMutex
 
 	// pollBastionStatusInterval is the time in-between status checks on the bastion object.
 	pollBastionStatusInterval = 5 * time.Second
@@ -946,8 +948,15 @@ func sshCommandLine(o *SSHOptions, bastionAddr string, nodePrivateKeyFiles []str
 	return connectCmd
 }
 
+func getKeepAliveInterval() time.Duration {
+	keepAliveIntervalMutex.RLock()
+	defer keepAliveIntervalMutex.RUnlock()
+
+	return keepAliveInterval
+}
+
 func keepBastionAlive(ctx context.Context, gardenClient client.Client, bastion *operationsv1alpha1.Bastion, stderr io.Writer) {
-	ticker := time.NewTicker(keepAliveInterval)
+	ticker := time.NewTicker(getKeepAliveInterval())
 	defer ticker.Stop()
 
 	for {

